Updated Graph API Doc for Teams user usage

Zhiliqiao888 opened the pull request:
1: Add new properties to existing getTeamsUserActivityTotalCounts API. 
Expose more metric properties such as 'MeetingsOrganized' and 'MeetingsAttended' to provide clients with more activity usage information.
2: Add new properties to existing getTeamsUserActivityCounts API. 
Expose more metric properties such as 'MeetingsOrganized' and 'MeetingsAttended' to provide clients with more activity usage information.
3: Add more properties to existing getTeamsUserActivityUserDetail API if the tenants with Preview Tenant List. 
Expose more metric properties such as 'HostTenantDisplayName' and 'ResourceTenantDisplayName' to provide clients with more activity usage information if the customer is within Teams Shared Channel Preview tenant list.
4: Add new API getTeamsUserActivityTotalDistributionCounts. 
Expose API for time range distribution user activity usage report.

opbld33 commented:
Docs Build status updates of commit _[1396071](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/13960710380a2d762664b3c9d17dcfe0980f5e1d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C23%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202202230823014302-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-26T08%3A25%3A02Z&sr=b&sp=r&sig=AK%2BIBiQu1TQmu4bf1nsPx87%2BUUMhXUddpgihrkkj%2FC0%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Lauragra commented:
@Zhiliqiao888 , please follow the get started steps in the wiki to set yourself up to be a contributor to the documentation so that you can follow our workflow process, set labels, etc. Thanks.

opbld34 commented:
Docs Build status updates of commit _[ccc2a23](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/ccc2a23e7f122c97780e1d3ed98088df86f04cda)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203160625241008-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-16T06%3A27%3A21Z&sr=b&sp=r&sig=J3I87bj7HCrNTa0gzvLkx5PvP%2BtQoJEJe%2F2OC8tF50w%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[c485efc](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/c485efcae553fa0a595e3509a784eeb60d56c42c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C17%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203170316158013-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-17T03%3A18%3A19Z&sr=b&sp=r&sig=8R1J8dbz1lNy2mUI4OC2ia9OnNzHcfJntRmiBtoN2LM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Zhiliqiao888 commented:
@Lauragra Could you start review? Customer is waiting for this doc change. Thank you.

JarbasHorst commented on changelog/Microsoft.O365Reporting.json:
```diff
@@ -1,5 +1,55 @@
+		"ChangeList": [
+		  {
+			  "Id": "9cabd554-398b-4dd4-8a36-5fe46c7cda94",
+			  "ApiChange": "Column",
```
Global: Column is not on the list of approved values for ApiChange. Please change this to "Method".

Zhiliqiao888 replied:
Done

JarbasHorst commented on changelog/Microsoft.O365Reporting.json:
```diff
@@ -1,5 +1,55 @@
+			"ApiChange": "Method",
+			"ChangedApiName": "getTeamsTeamActivityCounts",
+			"ChangeType": "Addition",
+			"Description": "Added support for the [getTeamsTeamActivityCounts](https://docs.microsoft.com/en-us/graph/api/reportroot-getTeamsTeamActivityCounts?view=graph-rest-beta) method.",
```
This API topic isn't part of this PR. Please, either include this API topic to this PR or remove this entry from this changelog record.

Zhiliqiao888 replied:
Removed

JarbasHorst commented:
Hello @Zhiliqiao888, I reviewed your PR and left some comments. Please, take a look when you will have time.

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+---
+title: "reportRoot: getTeamsUserActivityTotalDistributionCounts"
+description: "Get total distribution counts of Microsoft Teams user activity in a specific time period."
```
@JarbasHorst They are the similar ones. Total distribution counts means this API is to get aggregated count per activity type of all users within this tenant within specified time period.

JarbasHorst replied:
Thanks for the clarification!

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+| Delegated (personal Microsoft account) | Not supported.                           |
+| Application                            | Reports.Read.All                         |
+
+**Note**: For delegated permissions to allow apps to read service usage reports on behalf of a user, the tenant administrator must have assigned the user the appropriate Azure AD limited administrator role. For more details, see [Authorization for APIs to read Microsoft 365 usage reports](/graph/reportroot-authorization).
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+
+## Function parameters
+
+In the request URL, provide one of the following parameters with a valid value.
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+
+| Parameter | Type   | Description                              |
+| :-------- | :----- | :--------------------------------------- |
+| period    | string | Specifies the length of time over which the report is aggregated. The supported values for {period_value} are: D7, D30, D90, and D180. These values follow the format D*n* where *n* represents the number of days over which the report is aggregated. |
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+| Parameter | Type   | Description                              |
+| :-------- | :----- | :--------------------------------------- |
+| period    | string | Specifies the length of time over which the report is aggregated. The supported values for {period_value} are: D7, D30, D90, and D180. These values follow the format D*n* where *n* represents the number of days over which the report is aggregated. |
+
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+| :-------- | :----- | :--------------------------------------- |
+| period    | string | Specifies the length of time over which the report is aggregated. The supported values for {period_value} are: D7, D30, D90, and D180. These values follow the format D*n* where *n* represents the number of days over which the report is aggregated. |
+
+This method supports the `$format`, `$top`, and `$skipToken` [OData query parameters](/graph/query-parameters) to customize the response. The default output type is text/csv. However, if you want to specify the output type, you can use the OData $format query parameter set to text/csv or application/json.
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+---
+title: "reportRoot: getTeamsUserActivityTotalDistributionCounts"
```
![image](https://user-images.githubusercontent.com/96765315/159447557-6cffefaa-1193-4b8c-b670-8fd7dab3f945.png)
Seems it is already there, could you confirm if it is what you want?

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,201 @@
+---
+title: "reportRoot: getTeamsUserActivityTotalDistributionCounts"
```
Done

opbld30 commented:
Docs Build status updates of commit _[ede56c9](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/ede56c982dc507d2305e83a3d8d57ecf1a06218a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/resources/microsoft-teams-user-activity-reports.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/resources/microsoft-teams-user-activity-reports.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203220954029752-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-22T09%3A56%3A19Z&sr=b&sp=r&sig=gh95t0DHCZdyFdTycWAu%2FVIB2as%2FIz3tYS05HQ%2FqIOM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Zhiliqiao888 commented:
> Hello @Zhiliqiao888, I reviewed your PR and left some comments. Please, take a look when you will have time.

Thank you @JarbasHorst New files are checked in based on your comments.

JarbasHorst commented on api-reference/beta/toc.yml:
```diff
@@ -13738,6 +13738,8 @@ items:
             href: api/reportroot-getteamsuseractivitydistributionusercounts.md
           - name: Get distribution total user counts
             href: api/reportroot-getteamsuseractivitydistributiontotalusercounts.md
+          - name: Get total distribution counts of Microsoft Teams user activity
```
For simplicity and consistency, please replace "Get total distribution counts of Microsoft Teams user activity" with

- Get distribution total activity counts

OR

- Get total distribution activity counts

JarbasHorst commented on api-reference/beta/resources/microsoft-teams-user-activity-reports.md:
```diff
@@ -26,5 +26,5 @@ Use the Microsoft Teams user activity reports to get insights into the Microsoft
 | [Get distribution user counts](../api/reportroot-getteamsuseractivitydistributionusercounts.md) | Stream      | Get the number of Microsoft Teams licensed users by activity type over the selected period. The activity types are number of teams chat messages, private chat messages, calls, and meetings. |
 | [Get distribution total user counts](../api/reportroot-getteamsuseractivitydistributiontotalusercounts.md) | Stream      | Get the number of Microsoft Teams licensed or non-licensed users by activity type over the selected period. The activity types are number of teams chat messages, private chat messages, calls, and meetings. |
-
+| [Get total distribution counts of Microsoft Teams user activity](../api/reportroot-getTeamsUserActivityTotalDistributionCounts.md) | Stream      | Get the number of Microsoft Teams user activities over the selected period. Activities are team chat messages, private chat messages, calls, meetings, meetings organized, meetings attended, audio duration, video duration, screen share duration, post  ...
```
1) Please, use either "Get distribution total activity counts" or "Get total distribution activity counts" as the name of this method.

2) For consistency, "reportroot-getTeamsUserActivityTotalDistributionCounts.md" should be in lower case format.

3) Please, add a "comma" before "and".

JarbasHorst commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,207 @@
+---
+title: "reportRoot: getTeamsUserActivityTotalDistributionCounts"
+description: "Get total distribution counts of Microsoft Teams user activity in a specific time period."
```
For consistency with the description provided [here ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/15948/files#diff-6f9abba751fe7142fc11c2d154f6dc91aeb68aaed2a3f90dd3bc06a722e5950c)and because that description is more informative and clear, I recommend you to rephrase the descriptions used in lines **3 and 15**. Suggested revision:

- Get the number of Microsoft Teams user activities over the selected period. The activity types are team chat messages, private chat messages, calls, meetings, meetings organized, meetings attended, audio duration, video duration, screen share duration, post messages, and reply messages.

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,207 @@
+| Delegated (personal Microsoft account) | Not supported.                           |
+| Application                            | Reports.Read.All                         |
+
+**Note**: For delegated permissions to allow apps to read service usage reports on behalf of a user, the tenant administrator must have assigned the user the appropriate Azure Active Directory limited administrator role. For more details, see [Authorization for APIs to read Microsoft 365 usage reports](/graph/reportroot-authorization).
```
For consistency, please add ">" before \*\*Note\*\* => 

```
> **Note**: For...
```

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
````diff
@@ -0,0 +1,207 @@
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /reports/getTeamsUserActivityTotalDistributionCounts(period='D7')
````
For clarity and consistency, please replace "D7" with the correct placeholder "{period_value}".

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,207 @@
+
+| Parameter | Type   | Description                              |
+| :-------- | :----- | :--------------------------------------- |
+| period    | string | Specifies the length of time over which the report is aggregated. The supported values for {period_value} are: `D7, D30, D90, and D180`. These values follow the format D*n* where *n* represents the number of days over which the report is aggregated. Required.|
```
For consistency, please replace \`D7, D30, D90, and D180\` with \`D7\`, \`D30\`, \`D90\`, and \`D180\`.

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/resources/microsoft-teams-user-activity-reports.md:
```diff
@@ -26,5 +26,5 @@ Use the Microsoft Teams user activity reports to get insights into the Microsoft
 | [Get distribution user counts](../api/reportroot-getteamsuseractivitydistributionusercounts.md) | Stream      | Get the number of Microsoft Teams licensed users by activity type over the selected period. The activity types are number of teams chat messages, private chat messages, calls, and meetings. |
 | [Get distribution total user counts](../api/reportroot-getteamsuseractivitydistributiontotalusercounts.md) | Stream      | Get the number of Microsoft Teams licensed or non-licensed users by activity type over the selected period. The activity types are number of teams chat messages, private chat messages, calls, and meetings. |
-
+| [Get total distribution counts of Microsoft Teams user activity](../api/reportroot-getTeamsUserActivityTotalDistributionCounts.md) | Stream      | Get the number of Microsoft Teams user activities over the selected period. Activities are team chat messages, private chat messages, calls, meetings, meetings organized, meetings attended, audio duration, video duration, screen share duration, post  ...
```
1) Please, use either "Get distribution total activity counts" or "Get total distribution activity counts" as the name of this method.

2) For consistency, "reportroot-getTeamsUserActivityTotalDistributionCounts.md" should be in lower case format.

3) Please, add a "comma" before "and".

4) For consistency, please replace "Activities are" with "The activity types are".

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/toc.yml:
```diff
@@ -13738,6 +13738,8 @@ items:
             href: api/reportroot-getteamsuseractivitydistributionusercounts.md
           - name: Get distribution total user counts
             href: api/reportroot-getteamsuseractivitydistributiontotalusercounts.md
+          - name: Get total distribution counts of Microsoft Teams user activity
```
For simplicity and consistency, please replace "Get total distribution counts of Microsoft Teams user activity" with

- Get distribution total activity counts

OR

- Get total distribution activity counts

Zhiliqiao888 replied:
Done

JarbasHorst commented on api-reference/beta/toc.yml:
```diff
@@ -13738,6 +13738,8 @@ items:
           - name: Get distribution total user counts
             href: api/reportroot-getteamsuseractivitydistributiontotalusercounts.md
+          - name: Get total distribution counts of Microsoft Teams user activity
+            href: api/reportroot-getTeamsUserActivityTotalDistributionCounts.md
```
For consistency, "reportroot-getTeamsUserActivityTotalDistributionCounts.md" should be in lower case format.

Zhiliqiao888 replied:
Done

JarbasHorst commented on changelog/Microsoft.O365Reporting.json:
```diff
@@ -1,5 +1,47 @@
+			  "ApiChange": "Method",
+			  "ChangedApiName": "getTeamsUserActivityTotalCounts",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **Post Messages**, **Reply Messages**, **Audio Duration**, **Video Duration**, **Screen Share Duration**, **Meetings Organized**, **Meetings Attended** column to the [getTeamsUserActivityTotalCounts](https://docs.microsoft.com/en-us/graph/api/reportroot-getTeamsUserActivityTotalCounts?view=graph-rest-beta) method.",
```
Please, replace "column" with "columns". Same for lines 18 and 26.

Zhiliqiao888 replied:
Done.

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
```diff
@@ -0,0 +1,207 @@
+- Reply Messages
+- Report Period
+
+### Example 2: JSON output
```
Done

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md:
````diff
@@ -0,0 +1,207 @@
+Report Refresh Date,Team Chat Messages,Private Chat Messages,Calls,Meetings,Meetings Organized,Meetings Attended,Audio Duration,Video Duration,Screen Share Duration,Post Messages,Reply Messages,Report Period
+```
+
+### JSON
````
Done

opbld33 commented:
Docs Build status updates of commit _[8c85146](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/8c85146bdbf2d0e65fc2c30aa34b2e92093c7e31)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/resources/microsoft-teams-user-activity-reports.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/resources/microsoft-teams-user-activity-reports.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C23%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203230729163451-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-23T07%3A31%3A36Z&sr=b&sp=r&sig=c6pGr5MKkDt9C0OmwAfoFXhMqLpXngKTKh0DM0fn9Sk%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Zhiliqiao888 commented:
> Hello @Zhiliqiao888, I reviewed your PR and left some comments. Please, take a look when you will have time.

@JarbasHorst Thank you for the detailed review comments. I have updated.
And based on design change, I updated 2 attributes naming in reportroot-getteamsuseractivityuserdetail.md

opbld31 commented:
Docs Build status updates of commit _[38ed705](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/38ed7050fd7f0555d6fff74d6fcad9e27f5af24f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/resources/microsoft-teams-user-activity-reports.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/resources/microsoft-teams-user-activity-reports.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C23%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203230738441176-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-23T07%3A40%3A53Z&sr=b&sp=r&sig=tEB44AlTEHsulkg8dFhstHdsE2OsRuhkzaIupxw16ok%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JarbasHorst commented:
Hi @Zhiliqiao888, thanks again for addressing my previous comments. I left one additional comment. Please, take a look when you will have time. Also, FHL is taking place this week. Can we wait until next Monday with the approval (done by Lauragra) for this PR or is this topic urgent?

Zhiliqiao888 commented on api-reference/beta/api/reportroot-getteamsuseractivitycounts.md:
```diff
@@ -3,7 +3,7 @@ title: "reportRoot: getTeamsUserActivityCounts"
 description: "Get the number of Microsoft Teams activities by activity type. The activity types are number of teams chat messages, private chat messages, calls, and meetings. The activities are performed by Microsoft Teams licensed users."
```
Thank you @JarbasHorst  Done for this change.

opbld34 commented:
Docs Build status updates of commit _[c2267aa](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/c2267aa1c69432762ba084545972eb6ffed0f3f1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/resources/microsoft-teams-user-activity-reports.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/resources/microsoft-teams-user-activity-reports.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203290743159440-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-29T07%3A45%3A15Z&sr=b&sp=r&sig=M0WHBeuL7gpSACqPqG2%2F2HaoBmnKefqLKLATbCBOs88%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JarbasHorst commented:
@Lauragra this PR is ready for final review;

opbld30 commented:
Docs Build status updates of commit _[edb296d](https://github.com/Zhiliqiao888/microsoft-graph-docs/commits/edb296ddaed9adc25e26a538e006ec1f0787226e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getteamsuseractivitycounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitycounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotalcounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotalcounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivitytotaldistributioncounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitytotaldistributioncounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/api/reportroot-getteamsuseractivityuserdetail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivityuserdetail?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/resources/microsoft-teams-user-activity-reports.md](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/resources/microsoft-teams-user-activity-reports.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/reportroot-getteamsuseractivitycounts?view=graph-rest-beta&branch=pr-en-us-15948) (graph-rest-beta) |
[changelog/Microsoft.O365Reporting.json](https://github.com/Zhiliqiao888/microsoft-graph-docs/blob/users/zhiliqiao/teamsuserusage/changelog/Microsoft.O365Reporting.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202204070422518329-15948%5Cworkflow_report.html?sv=2020-08-04&se=2022-05-08T04%3A24%3A52Z&sr=b&sp=r&sig=fsqOEiBVo16sObo32uAjCq88V4w3LhDfAJcgp9Fx4Es%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
